cmd/juju/application/deploy.go

```diff
@@ -1200,6 +1202,7 @@ func (c *DeployCommand) maybeReadLocalBundle() (deployFn, error) {
 			ctx,
 			bundleDir,
 			bundleData,
+			nil,
```

Comment:
Perhaps a comment here like the filepath one below. just saying `// no bundleURL`